vignettes/item-deletion-examples.Rmd

```diff
@@ -0,0 +1,584 @@
+---
+title: "Applied examples"
```

Comment:
Suggest to be more descriptive: e.g., "Applied examples of item deletion statistics"